.travis.yml

```diff
@@ -30,15 +30,15 @@ jobs:
     - stage: "Tests"
       name: "Tests"
       script:
-        - yarn lint:hbs
-        - yarn lint:js
-        - yarn test
+        - npm run lint:hbs
+        - npm run lint:js
+        - npm test
```

Comment:
Interesting... guessing this change was from a codemod? Do you know what the reason for using `npm` over `yarn` is here?

Reply:
the codemod doesn't account for yarn/npm usage when updating the travis files. these should be `yarn` instead of `npm run`

Reply:
oh, no...was debugging something....will revert this